modules/ROOT/pages/6.0-release-notes-premium-changes.adoc

```diff
@@ -108,6 +211,69 @@ NOTE: There is an open https://github.com/tinymce/tinymce/discussions/7487[reque
 If a community-maintained version of the `paste` plugin becomes available, we will link to it from the xref:migration-from-5x.adoc#plugins-paste[_Migration Guide_].
 
 
+=== Real-time Collaboration
+
+The following improvements, changes, and fixes were made to Real-time Collaboration.
+
+Three additional options are now supported when Real-time Collaboration is active:
+
+* `valid_elements`
+* `valid_children`
+* `extended_valid_elements`
+
+Real-time Collaboration now supports the `autocompleter()` API.
+
+//xref:autocompleter.adoc[`autocompleter()`] ?
```

Comment:
@TheSpyder can you confirm these links and the inline boundaries below

Reply:
Neither of those references exist. I'll add some suggestions.

Reply:
After some discussion elsewhere in this PR, existing references and a ‘better approach’ was the way forward.